doc/src/odin/index_markdown.md

```diff
@@ -0,0 +1,451 @@
+
+
+Identifying/fixing issues
+------------------
+**Creating an Issue**
+
+   Odin II is still in development and it's possible there are bugs present. If Odin II doesn't perform as expected, or doesn't adhere to the verilog standard it is important to create an issue in the GitHub repository. There is a template included but make sure to include a micro-benchmark that reproduces the bug. This micro-benchmark should be as simple as possible. Also, include the expected outcome and the produced outcome. If this outcome requires additional input files please make sure those are included as well. Finally, it is important to link some documentation that provides insight on what Odin II is doing that differs from the the verilog standard. Linked below is a pdf of the IEEE Standard of Verilog (2005) that could help. 
+
+[IEEE Standard for Verilog Hardware Description Language](loghttp://staff.ustc.edu.cn/~songch/download/IEEE.1364-2005.pdf)
+
+  **Creating a Pull Request (PR)** 
+
+  **Important** Before creating a Pull Request (PR), if it is a bug you have happened upon and intend to fix make sure you create an issue beforhand.
+
+  Pull requests are intended to correct bugs and improve Odin's performance. To create a pull request, clone the vtr-verilog-rooting repository and branch from the master. Make changes to the branch that improve Odin and correct the bug. **Important** In addition to correcting the bug, it is required that test cases are created that reproduce the issue and included in the regression tests. An example of a good test case could be the micro-benchmark found in the "Issue" being addressed. Run through the regression tests (with yours included) to ensure that the modifications made correct the issue and don't cause other issues. Push these changes to the cloned repository and create the pull request. Add a description of the changes made (what file,line,etc) and reference the "issue" that it corrects. There is also a template provided on GitHub. 
+
+#### Creating a "Work in progress" (WIP) PR
+
+  A "work in progress" PR is a pull request that isn't complete or ready to be merged. It is intended to demonstrate that a said Issue is being addressed and indicates to others that they don't need to fix it. To create a WIP PR, clone the vtr-verilog-rooting repository and branch from the master. Make some changes and push them to the cloned repository. Create a pull request and **INCLUDE** WIP in the title. This will automatically indicate that this PR is not ready to be merged.  
+
+Help 
+-----
+
+If you have any questions or concerns there are multiple outlets to express them. 
+There is a google group for users who have questions that is checked regularly be Odin II team members:
+[VTR-USERS google group]
+
+
+
+
+CONTACT
+-------
+
+jamieson dot peter at gmail dot com ken at unb dot ca - We will service
```

Comment:
remove the contact section since the docs should indicate communication channels (github or vtr users)

Reply:
Ya I was gonna get rid of it and put it in the help, should I includ the google group for developers as well?

Reply:
yeah! thats good too :)